app/main/views/audits.py

```diff
@@ -180,6 +212,48 @@ def acknowledge_audit(audit_id):
     return jsonify(auditEvents=audit_event.serialize()), 200
 
 
+@main.route('/audit-events/<int:audit_id>/acknowledge-including-previous', methods=['POST'])
```

Comment:
But people never read documentation (until it's too late...)

If *you're* asked to do some ad-hoc data admin at some odd hour, do *you* stop and find the documentation for an endpoint that otherwise gives no clue that it does anything other than what it says on the tin?

I wouldn't even know where to look for that sort of documentation.